public/apps/configuration/constants.tsx

```diff
@@ -25,6 +25,8 @@ export const API_ENDPOINT_MULTITENANCY = API_PREFIX + '/multitenancy/tenant';
 export const API_ENDPOINT_TENANCY_CONFIGS = API_ENDPOINT + '/tenancy/config';
 export const API_ENDPOINT_SECURITYCONFIG = API_ENDPOINT + '/securityconfig';
 export const API_ENDPOINT_INTERNALUSERS = API_ENDPOINT + '/internalusers';
+export const API_ENDPOINT_INTERNALACCOUNTS = API_ENDPOINT + '/internalaccounts';
+export const API_ENDPOINT_SERVICEACCOUNTS = API_ENDPOINT + '/serviceaccounts';
```

Comment:
Do these endpoints exist in backend?

Based on the related PR https://github.com/opensearch-project/security/pull/2786/files, it seems like it is expecting query parameters `name` & `filterBy`.

Reply:
those endpoints in constants.tsx are actually the api endpoints from the backend of the dashboards's plugins that are called by the react front-end, not directly related to the endpoints specified on the the OS's plugin.

So the full chain is:

OS Dashboards FrontEnd -> OS Dashboards Backend -> OS Security's plugin.

Reply:
aah.. well learned something new today.